examples/bench/bench.cpp

```diff
@@ -11,6 +11,9 @@ struct whisper_params {
     int32_t what = 0; // what to benchmark: 0 - whisper encoder, 1 - memcpy, 2 - ggml_mul_mat
 
     std::string model = "models/ggml-base.en.bin";
+    std::string coreml_dir = "";
+    std::string openvino_dir = "";
+    bool disable_coreml = false;
```

Comment:
This is not a good change - we should not pass backend-specific info through the `libwhisper` API.

Reply:
--ov-e-device in Cli appears rather backend specific?